data/unk_80009410.data.s

```diff
@@ -9,6 +9,18 @@
 
 .balign 16
 
+# Unused. This appears to be RSP code, however it is not part of
+#         rspboot as rspbootTextEnd is at this symbol
+glabel D_800093F0
```

Comment:
Since this appears to be RSP code, I feel like it would be better to keep it in `rsp_boot.text.s` even if it's not part of `rspbootText` itself. (at least I don't think this file is appropriate for it since it's .data and it's already weird on its own)

Reply:
If anything I'd rather have another separate file, since eventually rspboot won't exist in it's current form and I'd rather not include this unrelated code in the eventual rspboot disassembly

Reply:
I personally feel like this belongs with the rest of the rsp code, even if it's unused/unidentified, but having another file would probably be okay as well